### PR TITLE
create course page for SAWSC training in Raleigh, NC

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: content/course-specific-material/introR-SAWSC-Feb17.Rmd [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Installation
+
+See [Before the Workshop](/intro-curriculum/Before) for information on what software should be installed prior to the course.
+
+### Tentative schedule
+
+**Day 1**
+
+-   08:30 am - 09:00 am -- Instructors available for questions
+-   09:00 am - 10:15 am -- [Introduction](/intro-curriculum/Introduction)
```
Comment (lindsaycarr) [inline content/course-specific-material/introR-SAWSC-Feb17.Rmd]: we will actually get started at 8:30, so instructors are available for questions 8-8:30. You can follow the Arkansas training pretty closely: https://owi.usgs.gov/R/training-curriculum/course-specific-material/introR-LMGWSCAR-Oct16/. There are a few sections that we don't do or we have yet to decide. 

Day 2 
01:00 pm - 02:30 pm – Visualize: ggplot2
02:45 pm - 04:00 pm – Visualize: swmrQW for censored data and publication-quality graphs

Both of these will be "Visualize: base R"

Day 3 
10:00 am - 10:30 am – The importance of reproducibility

We won't have this. Instead, the practice time would start at 10:00

File: content/course-specific-material/introR-SAWSC-Feb17.Rmd [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Download data from the [Data page](/intro-curriculum/data/).
+
+### Additional resources
+
+-   [USGS-R blog](https://owi.usgs.gov/blog/tags/r)
+-   [USGS-R twitter](https://twitter.com/USGS_R)
+-   [USGS-R GitHub](https://github.com/USGS-R) (package source code + bug/feature reporting)
+-   [RStudio cheatsheets](https://www.rstudio.com/resources/cheatsheets/) (data wrangling, visualization, shiny, markdown, RStudio, etc)
+-   [R Markdown](http://rmarkdown.rstudio.com/lesson-1.html)
+-   [Tutorial for updating packages when updating versions](https://www.datascienceriot.com/how-to-upgrade-r-without-losing-your-packages/kris/) (ask Scott for more info)
```
Comment (lindsaycarr) [inline content/course-specific-material/introR-SAWSC-Feb17.Rmd]: can you delete this one?

**Comment by ayan-usgs** [comment]
Requested changes have been implemented. You can give it another look if you want.

File: content/course-specific-material/introR-SAWSC-Feb17.Rmd [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+### Tentative schedule
+
+**Day 1**
+
+-   08:00 am - 08:30 am -- Instructors available for questions
+-   08:30 am - 10:00 am -- [Introduction](/intro-curriculum/Introduction)
+-   10:00 am - 10:15 am -- *Break*
+-   10:15 am - 12:00 pm -- [Get](/intro-curriculum/Get)
+-   12:30 am - 01:00 pm -- *Lunch*
```
Comment (lindsaycarr) [inline content/course-specific-material/introR-SAWSC-Feb17.Rmd]: this should be 12:00 pm, not 12:30 am :)

File: content/course-specific-material/introR-SAWSC-Feb17.Rmd [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+-   12:00 pm - 01:00 pm -- *Lunch*
+-   01:00 pm - 02:30 pm -- Visualize with base R: [Visualize](/intro-curriculum/Visualize/)
+-   02:30 pm - 02:45 pm -- *Break*
+-   02:45 pm - 04:00 pm -- Visualize with base R: [Visualize](/intro-curriculum/Visualize/) continued
+-   04:00 pm - 04:30 pm -- End of day wrap-up
+
+**Day 3**
+
+-   08:00 am - 08:30 am -- Instructors available for questions
+-   08:00 am - 10:00 am -- [Repeat](/intro-curriculum/Reproduce/)
```
Comment (lindsaycarr) [inline content/course-specific-material/introR-SAWSC-Feb17.Rmd]: this should be 8:30 am instead of 8:00 am

**Comment by lindsaycarr** [comment]
@ayan-usgs just made two comments about two of the times. Feel free to merge once those are fixed. Thanks for doing this!

**Pull request merged**